.github/DISCUSSION_TEMPLATE/new.yml

```
@@ -0,0 +1,69 @@
+name: New Discussion
+description: Please fill out the following information to create a new discussion.
+  Make sure to provide clear and concise details to help others understand and address your concerns effectively.
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Problem description
+      description: A clear and concise description of what the problem is
```

Maybe the phrasing is too restrictive here? It could also be an idea, for example.
```suggestion
      label: Idea/Problem description
      description: A clear and concise description of what the idea/problem is
```

---

I have updated this based on your suggestion